### DISCUSSION TITLE
Prowler V3 AllowList not working

### DISCUSSION THREAD

**Discussion by am-avios** [prowler-cloud/prowler#1628] [General]
Hi

I configured the allowlist.yaml as 

`Allowlist:
Accounts:
  "*":
  Checks:
    "sns_topics_kms_encryption_at_rest_enabled":
    Regions:
      - "eu-west-1"
    Resources:
      - "aws-controltower"
`

and scanned an account with prowler 
# prowler aws -p <> -R <> --region eu-west-1 -b -c sns_topics_kms_encryption_at_rest_enabled --allowlist-file allowlist.yaml

This still returned failed findings with resources name containing string 'aws-controltower'.

Can anyone point out what i am missing?

**Comment by jfagoagas** [comment]
Moved here: https://github.com/prowler-cloud/prowler/issues/1630

Thanks @am-avios
